Babylon/commands/api/solution/delete.py

```diff
@@ -20,7 +20,7 @@
 @require_platform_key("api_url")
 @pass_azure_token("csm_api")
 @option("--organization", "organization_id", type=QueryType(), default="%deploy%organization_id")
-@argument("solution_id", type=QueryType())
+@argument("solution_id", type=QueryType(), default="%deploy%solution_id")
```

Comment:
```suggestion
@argument("solution_id", type=QueryType())
```
I don't think setting a default to solution delete is a good idea, I prefer to be explicit on deletion
You can still use %deploy%solution_id if needed